app/src/main/res/layout/fragment_view_video.xml

```diff
@@ -38,6 +22,33 @@
         app:show_timeout="0"
         app:hide_on_touch="false" />
 
+    <ScrollView
+        android:id="@+id/mediaDescriptionScrollView"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_marginTop="?attr/actionBarSize"
+        android:layout_marginBottom="160dp"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintVertical_bias="0.0">
+
+        <TextView
+            android:id="@+id/mediaDescriptionTextView"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="#60000000"
+            android:hyphenationFrequency="full"
+            android:lineSpacingMultiplier="1.1"
+            android:padding="8dp"
+            android:textAlignment="center"
+            android:textColor="#eee"
+            android:textSize="?attr/status_text_medium"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintBottom_toBottomOf="parent"
```

Comment:
```suggestion
```

these don't do anything if the parent isn't a `ConstraintLayout`

Reply:
Resolved